2_delivery_care.do

```diff
@@ -71,7 +75,8 @@ c_sba_eff2_q
 			country_name == "Ghana2017" |
 			country_name == "Kiribati2018" |
 			country_name == "Montenegro2018" |
-			country_name == "Thailand2019" {;
+			country_name == "Thailand2019" |
+			country_name == "Tonga2019" {;
```

Comment:
When we use 21,23 and 31,33, it is typically assumed that we mean no special treatment of health facilities in Tonga's survey report. Would be nice of you to check this again on this first assignment.

Reply:
I checked the survey report; the delivery location in Tonga2019 is divided into private sector/public sector/home/other - page 130 and 131.

Reply:
You are right on th survey report not being specific enough. Ideally we want to see which particular sort of facilities being recognised as formal/skilled/qualified.
Good to go for Tonga.